bob-grpc/proto/bob.proto

```diff
@@ -79,3 +86,9 @@ message BobError {
     int32 code = 1; // Error code in case of error
     string desc = 2; // Error desription
 }
+
+// Delete operation options
+message DeleteOptions {
+    bool force_node = 1; // Force operation to be served by node to which it comes
+    bool with_aliens = 2; // Remove data in aliens as well
```

Comment:
I think, that we should always perform delete operation on aliens or at least inverse that flag, so that deleting in aliens will be enabled by default